### ISSUE TITLE
Dodging/Layering Issues

### ISSUE THREAD

**Issue by ToastyYogurtTime** [psifidotos/Latte-Dock#256]
In the most recent git build of Latte-Dock on AUR, I have the dock set to "Dodge All Active Windows" yet for windows from certain applications, it stays on top anyway. So far I have found that this affects Steam, Godot, and a couple games that run in full screen mode.
![2017-03-13_12 08 54](https://cloud.githubusercontent.com/assets/14959006/23866292/5f38d66a-07e6-11e7-8c9c-f99f95b14868.png)

Also, when compositing is turned off, the dock will stay on top of everything. I don't normally have compositing turned off but I do for some games and I have to close Latte so it doesn't block the screen. Can the dock be made less obtrusive when compositing is disabled?

**Comment by psifidotos** [comment]
> I have the dock set to "Dodge All Active Windows" yet for windows from certain applications, it stays on top anyway. So far I have found that this affects Steam, Godot, and a couple games that run in full screen mode.

that reminds me #80, we had added a workaround for this... I dont think we could blacklist all faulty apps that dont set their flags correctly... we will try to investigate in depth we this is happening and all these apps what flags are using wrongly..

@ToastyYogurtTime can you please use **xprop** or **xgrab** and upload here their output for some of the apps that create that issue?

> Also, when compositing is turned off, the dock will stay on top of everything. I don't normally have compositing turned off but I do for some games and I have to close Latte so it doesn't block the screen.

this is done because of the flags of our dock... 
@ToastyYogurtTime can you please check that if you set the dock to "AlwaysVisible" this issue doesnt exist?

**Comment by ToastyYogurtTime** [comment]
Here's the xprop output for the main Godot window:
```
`_NET_WM_NAME(UTF8_STRING) = "Godot Engine - Minilens - menu.tscn"
_NET_WM_ICON_GEOMETRY(CARDINAL) = 4294967249, 451, 46, 45
_NET_WM_ICON(CARDINAL) =        Icon (64 x 64):
                                                                        
                                                                        
                                                                        
                                ░░            ░░                        
                             ░░▒▒▒░          ░▒▒▒░░                     
                           ░▒▒▒▒▒▒▒          ▒▒▒▒▒▒▒░                   
                          ░▒▒▒▒▒▒▒▒░        ░▒▒▒▒▒▒▒▒░                  
                          ░▒▒▒▒▒▒▒▒▒        ▒▒▒▒▒▒▒▒▒░                  
                          ░▒▒▒▒▒▒▒▒▒░      ░▒▒▒▒▒▒▒▒▒░                  
                           ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒                   
                           ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒                   
                           ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒                   
                           ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒                   
                         ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░                 
               ░▒░      ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░      ░▒░       
               ▒▒▒░    ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░    ░▒▒▒       
              ░▒▒▒▒▒░ ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░ ░▒▒▒▒▒░      
             ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░     
             ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒     
            ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░    
           ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░   
           ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░   
            ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░    
             ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒     
             ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░     
              ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░      
               ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒▒▒░░  ░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░   ░░▒▒▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒▒░      ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░       ░▒▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒░        ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒         ░▒▒▒▒▒▒       
               ▒▒▒▒▒▒░   ░▒▒▒░  ░▒▒▒▒▒▒░░▒▒▒▒▒▒▒░  ▒▒▒▒░   ▒▒▒▒▒▒       
               ▒▒▒▒▒▒░  ░▒▒▒▒▒░ ░▒▒▒▒▒░   ▒▒▒▒▒▒  ▒▒▒▒▒▒   ░▒▒▒▒▒       
               ▒▒▒▒▒▒   ▒▒▒▒▒▒▒  ▒▒▒▒▒░   ░▒▒▒▒▒ ░▒▒▒▒▒▒▒  ░▒▒▒▒▒       
               ▒▒▒▒▒▒   ▒▒▒▒▒▒▒  ▒▒▒▒▒░   ░▒▒▒▒▒ ░▒▒▒▒▒▒▒  ░▒▒▒▒▒       
               ▒▒▒▒▒▒   ▒▒▒▒▒▒▒  ▒▒▒▒▒░   ░▒▒▒▒▒ ░▒▒▒▒▒▒▒  ░▒▒▒▒▒       
               ▒▒▒▒▒▒░  ▒▒▒▒▒▒▒ ░▒▒▒▒▒░   ░▒▒▒▒▒  ▒▒▒▒▒▒░  ░▒▒▒▒▒       
               ▒▒▒▒▒▒▒   ▒▒▒▒▒  ▒▒▒▒▒▒░   ░▒▒▒▒▒░ ░▒▒▒▒▒  ░▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒░   ░░░  ░▒▒▒▒▒▒░   ░▒▒▒▒▒▒░  ░░░   ▒▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒▒░      ░▒▒▒▒▒▒▒░   ░▒▒▒▒▒▒▒░      ░▒▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒▒▒▒░░░░▒▒▒▒▒▒▒▒▒░  ░▒▒▒▒▒▒▒▒▒▒░░░░▒▒▒▒▒▒▒▒▒       
               ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒       
                ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░       
            ░░   ▒▒▒▒░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░▒▒▒░   ░    
            ▒▒▒  ▒▒▒░    ░░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░     ▒▒▒░ ░▒▒    
            ▒▒▒  ▒▒▒░        ▒▒▒▒▒▒░░░░░░░░░░░▒▒▒▒▒░        ▒▒▒░ ░▒▒    
            ▒▒▒  ▒▒▒░ ░▒░░░  ▒▒▒▒▒░           ▒▒▒▒▒░  ░░▒▒  ▒▒▒░ ░▒▒    
            ░▒▒   ░░░ ░▒▒▒▒  ▒▒▒▒▒░  ░░░░░░░  ▒▒▒▒▒░ ░▒▒▒▒  ▒░   ░▒░    
            ░▒▒░      ░▒▒▒▒  ▒▒▒▒▒░ ░▒▒▒▒▒▒▒  ▒▒▒▒▒░ ░▒▒▒▒      ░▒▒░    
             ▒▒▒▒░░   ░▒▒▒▒  ░▒▒▒▒░ ░▒▒▒▒▒▒▒  ▒▒▒▒▒░ ░▒▒▒▒    ░▒▒▒▒     
             ░▒▒▒▒▒▒░░░▒▒▒▒      ░  ░▒▒▒▒▒▒▒  ░░     ░▒▒▒▒░░▒▒▒▒▒▒░     
              ░▒▒▒▒▒▒▒▒▒▒▒▒░░       ░▒▒▒▒▒▒▒        ░▒▒▒▒▒▒▒▒▒▒▒▒░      
               ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░░▒▒▒▒▒▒▒▒░░░░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒       
               ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░       
                ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒         
                  ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░          
                   ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░           
                     ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░             
                       ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░               
                         ░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░                 
                            ░░░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░░░                    
                                 ░░░░░░▒▒░░░░░░                         
                                                                        
                                                                        



XdndAware(ATOM) = BITMAP
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW
WM_CLASS(STRING) = "Godot_Editor", "Godot"
WM_NAME(STRING) = "Godot Engine - Minilens - menu.tscn"
_NET_WM_ALLOWED_ACTIONS(ATOM) = _NET_WM_ACTION_MOVE, _NET_WM_ACTION_RESIZE, _NET_WM_ACTION_MINIMIZE, _NET_WM_ACTION_MAXIMIZE_VERT, _NET_WM_ACTION_MAXIMIZE_HORZ, _NET_WM_ACTION_FULLSCREEN, _NET_WM_ACTION_CHANGE_DESKTOP, _NET_WM_ACTION_CLOSE
_KDE_NET_WM_FRAME_STRUT(CARDINAL) = 0, 0, 0, 0
_NET_FRAME_EXTENTS(CARDINAL) = 0, 0, 0, 0
_NET_WM_DESKTOP(CARDINAL) = 1
_KDE_NET_WM_ACTIVITIES(STRING) = "c13dd5ab-30da-424b-9400-64f7011f1e85"
WM_STATE(WM_STATE):
                window state: Normal
                icon window: 0x0
_NET_WM_STATE(ATOM) = _NET_WM_STATE_MAXIMIZED_VERT, _NET_WM_STATE_MAXIMIZED_HORZ
_KDE_NET_WM_USER_CREATION_TIME(CARDINAL) = 37973165`
```

And here's the Xprop output for Steam's Library window:
```
`_NET_WM_ICON_GEOMETRY(CARDINAL) = 4294967249, 451, 46, 45
_KDE_NET_WM_ACTIVITIES(STRING) = "00000000-0000-0000-0000-000000000000"
_NET_WM_ALLOWED_ACTIONS(ATOM) = _NET_WM_ACTION_MOVE, _NET_WM_ACTION_RESIZE, _NET_WM_ACTION_MINIMIZE, _NET_WM_ACTION_MAXIMIZE_VERT, _NET_WM_ACTION_MAXIMIZE_HORZ, _NET_WM_ACTION_FULLSCREEN, _NET_WM_ACTION_CHANGE_DESKTOP, _NET_WM_ACTION_CLOSE
_NET_WM_DESKTOP(CARDINAL) = 1
WM_STATE(WM_STATE):
                window state: Normal
                icon window: 0x0
_NET_WM_STATE(ATOM) = _NET_WM_STATE_MAXIMIZED_VERT, _NET_WM_STATE_MAXIMIZED_HORZ
WM_NAME(UTF8_STRING) = "Steam"
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW
WM_HINTS(WM_HINTS):
                Client accepts input or input focus: True
_NET_WM_ICON(CARDINAL) =        Icon (48 x 48):
                          ░░▒▒▓▓▓▓▒▒░░                  
                       ░▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒░               
                     ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░             
                   ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░           
                  ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒          
                ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░        
               ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░       
              ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░      
             ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▒▓▓▓▓▓▓▓▓░     
             ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░    ░░▓▓▓▓▓▓▓     
            ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒          ▒▓▓▓▓▓▒    
           ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░     ░░     ▒▓▓▓▓▓░   
           ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒   ░░▒▓▓▓░    ▒▓▓▓▓▓   
          ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░  ░▒▓▒░░▒▓▒░  ░▓▓▓▓▓▒  
          ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░  ░▓▒░    ░▓▒   ▒▓▓▓▓▓  
         ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░  ░▓░      ░▓░  ░▓▓▓▓▓░ 
         ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓   ▓▒        ▒▒  ░▓▓▓▓▓▒ 
         ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒  ░▓░        ░▓   ▓▓▓▓▓▓ 
        ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░  ░▓░        ░▓   ▓▓▓▓▓▓░
        ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒    ▒▒        ▒▒  ░▓▓▓▓▓▓▒
        ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░    ░▓░      ░▓░  ░▓▓▓▓▓▓▒
        ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░      ▒▓░    ░▓▒   ▒▓▓▓▓▓▓▒
        ░▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒        ▒▓▒▒▒▓▓▒   ░▓▓▓▓▓▓▓▓
           ░▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓          ░▒▒▒▒░    ▓▓▓▓▓▓▓▓▓
             ░▒▓▓▓▓▓▓▓▓▓▓▓░                   ▒▓▓▓▓▓▓▓▓▓
                ░▓▓▓▓▓▓▓▓▒                  ░▒▓▓▓▓▓▓▓▓▓▓
                  ░▒▓▓▒░░                ░░▒▓▓▓▓▓▓▓▓▓▓▓▒
                    ░                  ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒
                       ░▒▓▒░         ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒
                         ░▒▓░       ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░
                           ▒▓     ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ 
         ▒▒░                ▓░   ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒ 
         ░▓▓▒░              ▒▒  ▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░ 
          ▒▓▓▓▒▒            ▒▒ ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒  
          ░▓▓▓▓▓▓▒          ▒░ ░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░  
           ▒▒▒▒▒▒▓░        ░▓  ▒▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
           ░▒▒▒▒▒▓▒  ▒▒░ ░▒▓░ ░▓▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
            ▒▒▒▒▒▒▓░  ░▒▒▒▒░  ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
             ▒▒▒▒▒▒▒░        ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
             ░▒▒▒▒▒▒▒▒░   ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
              ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
               ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
                ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
                  ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
                   ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
                     ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
                       ░▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒░
                          ░░░▒▒▒▒▒▒░░░


_KDE_NET_WM_USER_CREATION_TIME(CARDINAL) = 38156393
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x2, 0x0, 0x0, 0x0, 0x0
WM_CLASS(STRING) = "Steam", "Steam"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
                user specified location: 183, 90
                user specified size: 1000 by 660
                program specified minimum size: 900 by 600
`
```

And yes, when I set the dock to "Always Visible," the compositing issue does not occur.

**Comment by psifidotos** [comment]
@ToastyYogurtTime until we can think this... you can do the following... 

go from Latte configuration and Tweak page to the Alternative Session. configure there your dock to be as you want it and set it to Always Visible.

So you can before disabling your compositing to switch to the alternative session... Please update also to latest master because there is a fix for a crash with (Alternative Session + !compositing)

**Comment by psifidotos** [comment]
@ToastyYogurtTime I have uploaded also a fix for visibility modes under !compositing... the docks now should become hidden... of course in hidden state 1px. line at the bottom is shown, for this I can not do anything more, it is the only way to restore dock on hovering...

**Comment by psifidotos** [comment]
@audoban concerning the windows issue.. is there a chance that kwin does not send signals for them? that would be the case if these windows have set "BypassWindowManagerHint". Should we create a blacklist for them, or do you think is something totally different?

**Comment by audoban** [comment]
I think that the problem is Godot, because this don’t have any window type configured, this issue can be fixed temporarily with the command xprop.

`xprop -f _NET_WM_WINDOW_TYPE 32a -set "_NET_WM_WINDOW_TYPE" "_NET_WM_WINDOW_TYPE_NORMAL"`

For latte I have treated this situation as a Net :: Normal window, because although it does not have any type if it has a geometry.

**Comment by psifidotos** [comment]
@audoban we can reverse the hypothesis around the window type. Currently we assume that apps are setting their flags correctly and we accept only these windows. The reality is far from this type... 

I propose all window types are accepted even the unknow, except those that have set their window types to the not accepted ,docks, menus, desktop etc...

It works for me just fine

**Comment by psifidotos** [comment]
@ToastyYogurtTime this is fixed in master, the xprop way is just an alternative temporary solution before the mentioned commit...

**Comment by audoban** [comment]
@ToastyYogurtTime please check the branch fix #256, do not worry you do not need xprop. The last commit should of that branch should fix the problem.
